sql/updates/world/2015_09_11_99_world_335.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `trinity_string`
+SET `content_default` = 'Object GUID is: %s'
```

Comment:
I saw a weird symbol showing up on the commit. No idea why it's showing though. Main question is, will both mysql and Heidi run it fine?